client/src/modules/patients/templates/action.cell.html

```diff
@@ -23,6 +23,11 @@
     </li>
     <li class="divider"></li>
     <li>
+      <a data-method="financialActivities" ng-href="reports/finance/financialPatient/{{row.entity.uuid}}" target="_blank">
+        <span class="fa fa-file-text-o"></span> <span translate>FORM.BUTTONS.PRINT_FINANCIAL_ACT</span>
```

Comment:
Since this opens a PDF report, let's make the icon a PDF file.  Also, since this isn't opening a print modal, a better name would be "View Financial Activity" instead of "Print".